src/public/app/widgets/ribbon_widgets/promoted_attributes.js

```diff
@@ -71,7 +71,7 @@ export default class PromotedAttributesWidget extends NoteContextAwareWidget {
         const promotedDefAttrs = note.getPromotedDefinitionAttributes();
 
         if (promotedDefAttrs.length === 0) {
```

Comment:
Issue with ribbon, combobox for "Book" looks strange:

<img width="185" alt="image" src="https://github.com/user-attachments/assets/c2b9b31a-b4b6-49e6-acfa-732c9d999c0d">

Compare with:

<img width="185" alt="image" src="https://github.com/user-attachments/assets/5d57c8bd-0563-4b7e-be0c-6a9f84ae401c">

Which also has an issue on hover:

<img width="185" alt="image" src="https://github.com/user-attachments/assets/f258bfd3-4f87-4f72-a8c5-fd8904f876f7">